code/game/gamemodes/vampire/vampire.dm

```diff
@@ -351,10 +354,25 @@ You are weak to holy things and starlight. Don't go into space and avoid the Cha
 		if(!(VAMP_SLAVE in vamp.powers))
 			vamp.powers.Add(VAMP_SLAVE)
 
-	// TIER 5
+	// TIER 5 (/vg/)
+	if(vamp.bloodtotal >= 400)
+		if(!(VAMP_MATURE in vamp.powers))
+			vamp.powers.Add(VAMP_MATURE)
```

Comment:
You can simplify this to ```vamp.powers |= VAMP_MATURE```, which will add it to powers if it's not there already